mypy/checkexpr.py

```diff
@@ -2892,19 +2891,19 @@ def check_super_arguments(self, e: SuperExpr) -> None:
                     if not isinstance(instance_type, (Instance, TupleType)):
                         # Too tricky, give up.
                         # TODO: Do something more clever here.
-                        self.chk.fail(messages.UNSUPPORTED_ARGUMENT_2_FOR_SUPER, e)
+                        self.chk.fail(messages.UNSUPPORTED_ARG_2_FOR_SUPER, e)
```

Comment:
I renamed this constant because I wanted to standardize all of them on the shorthand "ARG" for argument, since the constants can get fairly lengthy.